### DISCUSSION TITLE
New subscriber notifications to include name of newsletter

### DISCUSSION THREAD

**Discussion by thinkplaymake** [buttondown/discussions#44] [Q&A] [answered]
When you're a fan of buttondown, and use it for a number of different projects, it's hard to know which newsletter a "new subscriber" notification is referencing.

I love the notification emails, as it helps me keep track of activity, but as I'm running a few different newsletters, I'm having to check the "to:" header to see which account the notification was sent to.

So, my suggestion would be to modify the layout of the notification email, to include the newsletter name, i.e.

**From:**
![image](https://github.com/user-attachments/assets/f6527660-fe48-4914-85df-916970a9c423)

**To:** 
![image](https://github.com/user-attachments/assets/57df03ab-d243-4b53-8e6c-d4cea0fe8050)

Including the newsletter name in the subject line helps you quickly glance at the notification.
Including the newsletter name in the body of the email helps, when you're scanning the thread of notifications.

Would love this considered as a future addition!

**Comment by jmduke** [comment]
We do do this for accounts with multiple newsletters! Are you running multiple newsletters across multiple accounts?

**Reply by thinkplaymake** [reply]
Yes, this is across different accounts.

**Reply by jmduke** [reply, marked as answer]
Got it, yeah — this is working as intended to that extent. If you need differentiation between the notifications, I'd recommend combining them into a single account!
